### ISSUE TITLE
DayZ connection stopped working in Proton 9 beta 12

### ISSUE THREAD

**Issue by roadapathy** [ValveSoftware/Proton#7548]
# Compatibility Report
- Name of the game with compatibility issues: DayZ
- Steam AppID of the game: 221100 (223350?)

## System Information
- GPU: AMD RX 6700 TX
- Video driver version: OpenGL version string: 4.6 (Compatibility Profile) Mesa 23.2.1-1ubuntu3.1
- Kernel version: Linux version 6.8.0-rc6joel (gcc ( Ubuntu 13.2.0-4ubuntu3 ) 13.2.0, GNU ld (GNU Binutils for Ubuntu) 2.41 ) SMP PREEMPT_DYNAMIC
- and also Generic Ubuntu stock Kernel 6.5.0-21-generic
- Link to full system information report as [Gist](https://gist.github.com/): https://github.com/ValveSoftware/Proton/tree/proton_9.0
- Proton version: 9 beta 12

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
[steam-221100 - DayZ.log.tar.gz](https://github.com/ValveSoftware/Proton/files/14474477/steam-221100.-.DayZ.log.tar.gz)
- [x] that I have checked whether there are updates for my system available.

## Symptoms 
![dayzbugProton9beta12](https://github.com/ValveSoftware/Proton/assets/6241235/21ad087a-f580-4e20-bf9d-4dd9d5a047da)

It will not connect to the game but previous versions of Proton do connect. Symptom appears within Proton 9 beta 12 from Steam but also from the version that I compiled myself from the GIT page.

## Reproduction

This connection error appears in both the Generic Ubuntu Kernel and my own, which is the latest 6.8 mainline and both the Valve official Proton and the one that compiled from the GIT page.

**I would like to also add** that DayZ mods from **Steam Workshop** have never worked. I am forced to use an alternative launcher called DZGUI. The DZGUI launches Steam and somehow renames the mod files to where the game works. Otherwise, the game cannot use mods at all and Steam Workshop is useless- it becomes 100% vanilla DayZ.

**Comment by kisak-valve** [comment]
Hello @roadapathy, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to https://github.com/ValveSoftware/Proton/issues/3899#issuecomment-1975319804.

**Comment by roadapathy** [comment]
Thanks! I didn't check to see if it was officially supported. It runs so well in Proton overall that I just assumed it was officially supported.

**Comment by roadapathy** [comment]
> Hello @roadapathy, we're using one issue report per unofficially supported game title, so I've gone ahead and transferred this issue report to [#3899 (comment)](https://github.com/ValveSoftware/Proton/issues/3899#issuecomment-1975319804).

Kisak, I wanted to share that Proton 9 Beta 15 had in the update a mention of DayZ and that "DayZ is able to connect to servers again" but I'm not seeing this. I get the same error that it cannot connect. Maybe we're talking about two different things?

I tried Proton Beta 15 that I compiled and also the Steam version- the latest update that may or may not be Beta 15.
